src/test/resources/de/tum/cit/ase/ares/integration/testuser/securitypolicies/java/maven/archunit/aspectj/PolicySelfTestDefaultRestrictive.yaml

```
@@ -0,0 +1,37 @@
+# SecurityPolicyJavaDirector, ArchUnit and AspectJ, so the backend under test does not
+# change.
+thisPolicyFileCompliesToThePolicyVersion: 1
+regardingTheSupervisedCode:
+  theFollowingProgrammingLanguageConfigurationIsUsed: JAVA_USING_MAVEN_ARCHUNIT_AND_ASPECTJ
+  theSupervisedCodeUsesTheFollowingPackage: "de.tum.cit.ase.ares"
+  theMainClassInsideThisPackageIs: "Main"
+  theFollowingClassesAreTestClasses:
+    - "de.tum.cit.ase.ares.integration.testuser.SecurityUser"
+    - "de.tum.cit.ase.ares.integration.testuser.PackageAccessUser"
+  theFollowingResourceAccessesArePermitted:
+    regardingFileSystemInteractions: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)